west.yml

```diff
@@ -77,7 +77,7 @@ manifest:
       revision: 3776c158fe138a72c97c187e4d31c81c37884be9
       path: modules/crypto/mbedtls
     - name: mcuboot
-      revision: 54c1e3fb6ba4dcf7b315943bcfcda1e10a4042f5
+      revision: 09be78364430f48287fad54ff66c16485c93d372
```

Comment:
Another PR was merged, this needs updating

Reply:
Agreed; this is currently two behind master.